doc/configuration_options.rst

```diff
@@ -166,6 +166,12 @@ Description of common options
   master which is commonly used to upload artifacts to another host.
   This credential id is set in the buildfarm_deployment.
 
+* **Upload destination credential ID**: the ID of the credential entry managed
+  on the Jenkins master which contains the destination information used to
+  upload artifacts to another host.
+  This credential id is also set in the buildfarm_deployment.
```

Comment:
```suggestion
  This credential id is set in the buildfarm_deployment.
```

Is this `also` is in reference to the Credential ID above or to the fact that the buildfarm_deployment credential and this one must match? If the former, I think the also can be removed so the sentences in this paragraph and the one above are identical which would bother me in an essay but not in technical documentation.